Dockerfile

```diff
@@ -1,8 +1,13 @@
 FROM ruby:alpine
+MAINTAINER Chef Software, Inc. <docker@chef.io>
+
+ARG CHANNEL=stable
```

Comment:
If we are not using the CHANNEL arg, I don't think we need to keep it.

Reply:
Totally right - leftover from my testing of trying to just use the omnibus stuff. I'll kill it.